src/components/books/book.css

```diff
@@ -1,103 +1,331 @@
+:root {
+  /* --medium-blue: #307bbe; */
+  --bubble-gum-pink: #ff5ab7;
+  --black: #1e1e1e;
+  --pinkish-grey: #c7c7c7;
+  --very-light-pink: #c4c4c4;
+  --pale-grey: #f5f6fa;
+  --black-5: rgba(0, 0, 0, 0.05);
+  --z: #fafafa;
+  --black-two: #121212;
+  --white: #e8e8e8;
+  --medium-blue: #307bbe;
+  --cool-blue: #4386bf;
+  --azure: #0290ff;
+  --white-two: #fff;
+}
 
 .parent {
-  background-color: #eaebee;
   padding: 60px 60px;
   display: flex;
   flex-direction: column;
   gap: 20px;
+  box-shadow: 0 0 60px 0 var(--black-5);
+  background-color: var(--z);
 }
 
+/* book list  */
+
 .bookli {
-  background-color: #fff;
+  background-color: var(--white-two);
   padding: 20px 20px;
-  border: 1px solid #ccc9c9;
-  border-radius: 20px;
+  border-radius: 10px;
+  border: solid 1px var(--white);
+  display: flex;
+  justify-content: space-between;
+}
+
+.book-item-left {
+  display: flex;
+  flex-direction: column;
+}
+
+.category-name {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
 }
 
 .main {
   margin: 12px 0 12px 0;
 }
 
-.main h2 {
-  font-size: 30px;
+.book-title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.675rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
 }
 
 .bookAuthor {
-  font-size: 20px;
-  color: #bebcbc;
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.975rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
 }
 
-.nav {
+.list {
   display: flex;
-  text-decoration: none;
-  justify-content: space-around;
+  gap: 10px;
+}
+
+.comment {
+  width: 3.375rem;
+  height: 1.188rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.975rem;
+  font-weight: 300;
+  font-stretch: normal;
```

Comment:
- [ ] Please fix the overflow of your project the user should be able to see all the feature in your project without scrolling 
![image](https://user-images.githubusercontent.com/74744787/222922634-bbd16d1c-7c3d-4f15-bec9-909f8d3d5746.png)